pipeline.cpp

```diff
@@ -74,14 +74,22 @@ void DrawLine(Buffer2D<PIXEL> & target, Vertex* const triangle, Attributes* cons
 }
 
 /*************************************************************
- * DRAW_TRIANGLE
  * Finds the determinant.
  ************************************************************/
 double Determinant(int ax, int ay, int bx, int by)
 {
-    return (ax * by) - (ay * bx); 
+    return (ax * by) - (ay * bx);
 }
 
+double interpolate(double area, double det1, double det2, double det3, double c1, double c2, double c3)
+{
+    //Finding where the point is in the traingle and how much color is in each part
+    det1 /= area;
+    det2 /= area;
+    det3 /= area;
+
```

Comment:
I just used the approach from the reading, which it looks like you are doing but you are just passing in the determinants.